boards/arm/stm32l562e_dk/doc/index.rst

```diff
@@ -259,14 +259,30 @@ You should see the following message on the console:
 Building Secure/Non-Secure Zephyr applications with Arm |reg| TrustZone |reg|
 -----------------------------------------------------------------------------
 
-The TF-M integration sample :ref:`tfm_ipc` can be run on a STM32L562E-DK Discovery,
-using the ``stm32l562e_dk_ns`` target. When building a ``*_ns`` image with TF-M,
-a ``build/tfm/postbuild.sh`` bash script will be run automatically as a post-build step
-to make some required flash layout changes. The ``build/tfm/regression.sh`` script will
-need to be run to perform device initialization, and then run ``west flash --hex-file build/tfm_merged.hex``
-to flash the board.
-
-Check the ``build/tfm`` directory to ensure that the commands required by these scripts
+The TF-M integration sample :ref:`tfm_ipc` can be run on a ST Nucleo L552ZE Q.
+In TF-M configuration, Zephyr is run on the non-secure domain. A non-secure image
+could be generated using ``nucleo_l552ze_q_ns`` as build target.
```

Comment:
Repeat issue:

```suggestion
can be generated using ``nucleo_l552ze_q_ns`` as build target.
```

Reply:
Sure ... and I need to change the board name as well :-)